src/Sarif/Writers/SarifLogger.cs

```diff
@@ -462,6 +463,27 @@ public void AnalyzingTarget(IAnalysisContext context)
         {
         }
 
+        public void TargetAnalysisCompleted(IAnalysisContext context)
+        {
+        }
+
+        public void LogMemoryUsage(IAnalysisContext context)
+        {
+            var currentProcess = Process.GetCurrentProcess();
```

Comment:
>var currentProcess = Process.GetCurrentProcess(); [](http://example.com/codeflow?start=12&length=49)

nit: wrap this call in `using` as it allocates a lot resources

Reply:
fixed